xDrip/Services/CloudSync/NightscoutService.swift

```diff
@@ -141,7 +161,7 @@ final class NightscoutService {
         guard checkUseCellular(), checkSkipLANUploads() else { return }
```

Comment:
I would also like to see if the request was skipped because of cellular or wifi.